### PR TITLE
don't check diffed_files if PR into pinning, since rerender does nothing

### PR DESCRIPTION
**Pull request by CJ-Wright**

### CODE REVIEW

**Comment by CJ-Wright** [comment]
we need to clean up the graph as well

File: conda_forge_tick/auto_tick.py [hunk 1]
```diff
@@ -207,7 +207,7 @@ def run(
 
     # TODO: Better annotation here
     pr_json: typing.Union[MutableMapping, None, bool]
-    if isinstance(migrator, MigrationYaml) and not diffed_files:
+    if isinstance(migrator, MigrationYaml) and not diffed_files and feedstock_ctx.attrs['name'] != 'conda-forge-pinning':
```
Comment (beckermr) [inline conda_forge_tick/auto_tick.py]: Do we need to limit this just to the MigrationYaml and not the other migrators?

Comment (CJ-Wright) [reply conda_forge_tick/auto_tick.py]: Yes, rerender might not do anything for version updates or package replacements and that would be ok.

**Comment by beckermr** [comment]
LGTM!

**Pull request merged**